src/components/slider/slider.tsx

```diff
@@ -261,6 +273,10 @@ export class Slider {
         );
     }
 
+    public componentDidRender() {
```

Comment:
For some reason it doesn't work quite as well without this extra synchronisation step. Is the debounce missed, or the synchronisation step called too early as Adrian suggested?

In any case, this fixes it